loading/data/splign-manual/generate-loading-data

```diff
@@ -64,6 +64,7 @@ def parse_splign(fn, txdata):
         raise KeyError(f"{tx_ac}: no cds or hgnc info in {txdata_fn}")
 
     hgnc = txd["hgnc"]
```

Comment:
Perhaps we should rename "hgnc" to "gene_symbol" here to reduce confustion.

Reply:
Done. I kept txd["hgnc"] as is since we haven't made changes the the txinfo.yaml file yet.